trymito.io/pages/index.tsx

```diff
@@ -51,27 +51,27 @@ const Home: NextPage = () => {
             <div className={pageStyles.subsection}>
               <div className={homeStyles.functionality_text}>
                 <h1>
-                  <span className='text-highlight'> Explore </span> your data visually
+                  <span className='text-highlight'>Automate</span> analyses with Python
                 </h1>
                 <p className='display-mobile-only'>
-                  View CSVs, Excel files, and dataframes in an Excel-like interface. 
-                  Explore your data by graphing, viewing summary stats, and creating pivot tables.
+                  Every edit to the Mito spreadsheet automatically generates Python code. 
+                  Mito is the fastest way to fully automate a spreadsheet workflow.
                 </p>
                 <p className='display-desktop-only-inline-block'>
-                  View CSVs, Excel files, and dataframes in an Excel-like interface. 
+                  Every edit to the Mito spreadsheet automatically generates Python code. 
                 </p>
-                <p className='display-desktop-only-inline-block'> 
-                  Explore your data by graphing, viewing summary stats, and creating pivot tables.
+                <p className='display-desktop-only-inline-block'>
+                  Mito is the fastest way to fully automate a spreadsheet workflow.
                 </p>
-                <a href="https://docs.trymito.io/how-to/summary-statistics" target="_blank" rel="noreferrer" className={pageStyles.link_with_p_tag_margins}>
+                <a href="https://docs.trymito.io/how-to/using-the-generated-code" target="_blank" rel="noreferrer" className={pageStyles.link_with_p_tag_margins}>
```

Comment:
It would be awesome if we could link to a killer blog or case study here! Linking to a docs page whose only picture is an error is not the best marketing....